xbmc/windowing/gbm/WinSystemGbmGLESContext.cpp

```diff
@@ -42,6 +47,10 @@ std::unique_ptr<CWinSystemBase> CWinSystemBase::CreateWinSystem()
 bool CWinSystemGbmGLESContext::InitWindowSystem()
 {
   CLinuxRendererGLES::Register();
+#if HAS_GLES == 3
+  CLinuxRendererGLES3::Register();
+#endif
+
```

Comment:
This will result in:
CLinuxRendererGLES::Register();
CLinuxRendererGLES3::Register();
Is this intended?